packages/react-core/src/components/Drawer/Drawer.tsx

```diff
@@ -15,6 +15,10 @@ export interface DrawerProps extends React.HTMLProps<HTMLDivElement> {
   isStatic?: boolean;
   /** Position of the drawer panel */
   position?: 'left' | 'right';
+  /** Lifecycle function invoked when the drawer has been mounted to the DOM. */
+  onMount?: () => void;
+  /** Lifecycle function invoked when the drawer has been unmounted from the DOM. */
+  onUnMount?: () => void;
```

Comment:
Can you just call this `unMount` to [match React?](https://reactjs.org/docs/react-component.html#unmounting)